bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -144,38 +144,42 @@
 
     private final Object stateLock = new Object();
 
+    private final Object jobLock = new Object();
+
     private final SonosStateDescriptionOptionProvider stateDescriptionProvider;
 
     private final Runnable pollingRunnable = () -> {
```

Comment:
While you are changing code here anyway: Can you convert the runnable to a `private void poll()`?

Reply:
Done